app/forms/collection_edit_form.rb

```diff
@@ -21,6 +27,8 @@ def self.model_attributes(attrs)
     attrs.dup.each do |key, value|
       if value.is_a? Array
         attrs[key].map! { |v| to_hash(v) }
+      elsif value.nil? || value.to_s.strip.blank?
```

Comment:
@mcritchlow I just try to apply `#compact` in https://github.com/ucsdlib/horton/blob/feature/identifier/app/forms/collection_edit_form.rb#L7 `attrs = super(attrs).compact` but I got a weird undefined method `with_indifferent_access' for "0":String